tools/schematics/testing.ts

```diff
@@ -275,6 +275,9 @@ async function publishAllPackages(): Promise<void> {
 
   /** Number of packages published, successfully or not */
   let completedCount = 0;
+  /** Transform storefrontstyles and dist to use symbolic paths before publish */
+  execSync("ts-node tools/scripts/regex-scss-path-replace.ts ./dist");
+  execSync("ts-node tools/scripts/regex-scss-path-replace.ts ./projects/storefrontstyles");
```

Comment:
[medium]
1. Ideally, let's to run the new regex-replace as part of the regular nx `build` task of each library.
Thanks to this it won't matter from where you'll build or publish libs.
This script `tools/schematics/testing.ts` is used only for internal testing.

2. if we make changes inside the folder `storefrontstyles`, then we're changing the sources, which is a bummer. I believe, instead we should copy those styles to /dist folder (as an "artificial build" step), and then regex-replace only there in /dist folder